RubberduckTests/Grammar/VBAParserTests.cs

```diff
@@ -2614,7 +2614,7 @@ Sub Test()
     Debug.Print 1,
 End Sub";
             var parseResult = Parse(code);
-            AssertTree(parseResult.Item1, parseResult.Item2, "//debugPrintStmt/outputList");
+            AssertTree(parseResult.Item1, parseResult.Item2, "//lExpression/outputList");
```

Comment:
`printMethod` looks for the `Print` part. This looks for the `outputList` child of the `objectPrintExpr`. Since the setup does not allow to evaluate whether a child of a named alternative is present, `lExpression` is used for the `objPrintExpr` part.